packages/main/src/Calendar.js

```diff
@@ -61,26 +61,56 @@ const metadata = {
 			multiple: true,
 		},
 
+		/**
+		 * Determines the мinimum date available for selection.
+		 *
+		 * @type {String}
+		 * @defaultvalue undefined
+		 * @public
+		 */
+		minDate: {
+			type: String,
+			defaultValue: undefined,
```

Comment:
Is it necessary to set "undefined" here? Isn't it undefined by default (or is it empty string)?

Reply:
It's an empty string, but still a falsy value so it can be removed, thanks.